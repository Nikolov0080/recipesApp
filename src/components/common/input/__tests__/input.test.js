import React from 'react'
import ReactDOM from 'react-dom';
import Input from '../input';
import {  cleanup } from '@testing-library/react';
import '@testing-library/jest-dom/extend-expect';
import renderer from 'react-test-renderer';

afterEach(cleanup);

it('renders input OK', () => {
    const div = document.createElement('div');
    ReactDOM.render(<Input />, div);
    ReactDOM.unmountComponentAtNode(div);
});

it('matches snapshot', () => {
    const three = renderer.create(<Input label='test' name='test' type='text' />).toJSON();
    expect(three).toMatchSnapshot()
})

it('matches snapshot', () => {
    const three = renderer.create(<Input label='skillLevel' name='test' type='text' />).toJSON();
    expect(three).toMatchSnapshot()
})


