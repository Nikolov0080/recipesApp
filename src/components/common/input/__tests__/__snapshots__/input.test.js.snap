// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot 1`] = `
<div
  data-testid="regInput"
>
  <br />
  <input
    className="input_text"
    name="test"
    onChange={[Function]}
    placeholder="test"
    type="text"
  />
</div>
`;

exports[`matches snapshot 2`] = `
<div
  data-testid="regInput"
>
  <br />
  <input
    className="input_text"
    name="test"
    onChange={[Function]}
    placeholder="skillLevel"
    type="text"
  />
</div>
`;
